src/qml/filters/deband/ui.qml

```diff
@@ -0,0 +1,440 @@
+            filter.set(directionParam, directionDefault)
+            filter.set(blurParam, blurDefault)
+            filter.set(couplingParam, couplingDefault)
+            filter.savePreset(allParams, qsTr('FFmpeg default values'))
+
+            // Custom preset
+            filter.set(thr1Param, 0.0150291)
+            filter.set(thr2Param, 0.03994)
+            filter.set(thr3Param, 0.0150291)
+            filter.set(thr4Param, 0.0150291)
+            filter.set(linkParam, false)
+            filter.set(rangeParam, 90)
+            filter.set(directionParam, directionMax)
+            filter.set(blurParam, true)
+            filter.set(couplingParam, false)
+            filter.savePreset(allParams, qsTr('Blue gradient in open sky'))
+
+            // Custom preset
+            filter.set(thr1Param, 0.0150291)
+            filter.set(thr2Param, 0.0150291)
+            filter.set(thr3Param, 0.03994)
+            filter.set(thr4Param, 0.0150291)
+            filter.set(linkParam, false)
+            filter.set(rangeParam, 44)
+            filter.set(directionParam, directionMax)
+            filter.set(blurParam, true)
+            filter.set(couplingParam, false)
+            filter.savePreset(allParams, qsTr('Red gradient in open sky'))
+
+            // Custom preset
+            filter.set(thr1Param, 0.0100294)
+            filter.set(thr2Param, 0.0100294)
+            filter.set(thr3Param, 0.0100294)
+            filter.set(thr4Param, 0.0100294)
+            filter.set(linkParam, true)
+            filter.set(rangeParam, 23)
+            filter.set(directionParam, directionMax)
+            filter.set(blurParam, true)
+            filter.set(couplingParam, false)
+            filter.savePreset(allParams, qsTr('Full range to Limited range'))
+
+            // Default preset
+            // Same as Full to Limited preset
+            filter.savePreset(allParams)
+        }
+        filter.blockSignals = false
+
+        setControls()
+    }
+
+
+    width: 350
+    height: 450
+
+
+    GridLayout {
+        columns: 3
+        anchors.fill: parent
+        anchors.margins: 8
+
+        // Row split
+
+        Label {
+            text: qsTr('Preset')
+            Layout.alignment: Qt.AlignRight
+        }
+        Shotcut.Preset {
+            id: idPreset
+            Layout.columnSpan: 2
+            parameters: allParams
+            onPresetSelected: setControls()
+        }
+
+        // Row split
+
+        Label {
+            text: qsTr('Y/R Threshold')
+            Layout.alignment: Qt.AlignRight
+        }
+        Shotcut.SliderSpinner {
+            id: idThr1
+            minimumValue: 0
+            maximumValue: 100
+            decimals: 1
+            suffix: ' %'
+            onValueChanged: setThreshold(thr1Param, value)
+        }
+        Shotcut.UndoButton {
+            onClicked: idThr1.value = thrToPct(thr1Default)
+        }
+
+        // Row split
+
+        Label {
+            text: qsTr('Cb/G Threshold')
```

Comment:
Would be nice to get some tool tips on some of these. Will normal users know what "Cb/G" means?

Reply:
I agree and thought a lot about this.  I decided to try making the labels friendly to novices, then put some techie stuff in the tool tips.
My theory is that most people wouldn't use a Deband filter in RGB mode anyway (no range compression from RGB sources), which frees the labels to focus on YCbCr mode.
I'm stretching "luma" to mean "contrast" in this context, but functionally it works.  The higher the contrast in the luma plane, the higher the "contrast" threshold must be to count as banding.
The other labels are hopefully self-explanatory.